pandas/tests/frame/test_missing.py

```diff
@@ -739,8 +739,6 @@ def test_interp_rowwise(self):
         expected[4] = expected[4].astype(np.float64)
         assert_frame_equal(result, expected)
 
-        # scipy route
-        tm._skip_if_no_scipy()
```

Comment:
I intentionally did not replace this with a decorator because it looks unnecessary (none of the subsequent code requires spicy)

Reply:
ok